tests/v0_tests/test_get_indexes.py

```diff
@@ -44,15 +20,15 @@ def test_get_indexes(self):
         instance.
         """
         ix_0 = self.client.get_indexes()
-        assert not self._is_index_name_in_get_indexes_response(self.index_name_1, ix_0)
+        assert not self._is_index_name_in_get_indexes_response(self.generic_test_index_name, ix_0)
 
```

Comment:
ditto on the weird pattern of reassigning the `self.text_index...` 

we shouldn't be reassigning object attributes unless we need to